Include/object.h

```diff
@@ -235,8 +235,12 @@ PyAPI_FUNC(void *) PyType_GetModuleState(struct _typeobject *);
 
 /* Generic type check */
 PyAPI_FUNC(int) PyType_IsSubtype(PyTypeObject *, PyTypeObject *);
-#define PyObject_TypeCheck(ob, tp) \
-    (Py_IS_TYPE(ob, tp) || PyType_IsSubtype(Py_TYPE(ob), (tp)))
+
+PyAPI_FUNC(int) PyObject_TypeCheck(PyObject *, PyTypeObject *);
```

Comment:
Hum, I don't why you expect the function in the dynamic libpython library, whereas it's only implemented as a static inline function. Please remove this definition.

Reply:
Noted. Thanks!